server/src/main/java/org/spine3/server/storage/memory/InMemoryCommandStorage.java

```diff
@@ -104,4 +102,10 @@ private void put(CommandId id, CommandStorageRecord record) {
     private CommandStorageRecord get(CommandId id) {
         return storage.get(id);
     }
+
+    private void checkNotContains(CommandId id, String commandId) {
+        if (storage.containsKey(id)) {
```

Comment:
This hints on the fact that we need a separate method `update()`, which would take the same arguments, but assume that there's a record with such ID.